mindsdb/api/mysql/mysql_proxy/executor/executor_commands.py

```diff
@@ -694,12 +694,11 @@ def answer_describe_predictor(self, statement):
         # try full name
         attribute = None
         model_info = self._get_model_info(statement.value)
+        parts = statement.value.parts.copy()
+        attribute = parts.pop(-1)
+        model_info = self._get_model_info(Identifier(parts=parts)) if not model_info  else model_info
```

Comment:
here two _get_model_info are called in sequence. 
The right way is: to call once,  and call once again with different name only if model wasn't found.
(As it was before)
Try this PR #6686 (or update from staging, because it was merged long ago) - it should help you